cmd/kubeadm/app/constants/constants.go

```diff
@@ -31,6 +31,9 @@ const (
 	APIServerKubeletClientCertName           = "apiserver-kubelet-client.crt"
 	APIServerKubeletClientKeyName            = "apiserver-kubelet-client.key"
 
+	ServiceAccountKeyBaseName = "sa"
+	ServiceAccountKeyName     = "sa.key"
```

Comment:
Should split private/public portions into separate files. The apiserver should get the public portion.